lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart

```diff
@@ -47,6 +48,12 @@ class GuessSecretWordCubit
         );
       }
       _hasSuccess = true;
+      _pressedOptions = [
```

Comment:
Does this not also include the succes button? Or is that one still clickable? :)

Reply:
Still clickeable ;) (which is preferred)